src/graphics/program-lib/programs/basic.js

```diff
@@ -133,15 +133,16 @@ var basic = {
             code += "   alphaTest(gl_FragColor.a);\n";
         }
 
-        if (options.pass === SHADER_PICK) {
+        if (options.pass !== SHADER_PICK) {
             // ##### PICK PASS #####

```

Comment:
this "PICK PASS" comment should removed, as that's the empty branch of code you removed

Reply:
yes - I will remove

Reply:
you could also structure it like this
```
if (options.pass === SHADER_DEPTH) {
} else if (options.pass !== SHADER_PICK) {
}
```

Reply:
removed